### ISSUE TITLE
Choose annotation name from dropdown

### ISSUE THREAD

**Issue by DVassilev** [CSAILVision/LabelMeAnnotationTool#42]
Is it possible to define the names that can be used for annotating so they can be choose from a dropdown?

**Comment by Abhijit-2592** [comment]
Yes. It is clearly mentioned in the readme section add &objects=car,person,building in your URL and it will show you the drop down when you label them

**Comment by dobkeratops** [comment]
see also #54 ; you could combine with this issue, and make the dropdown menu default to the last label that you used, to save clicks when highlighting 'all the people', 'all the cars', 'all the trees' etc
